src/main/java/calculator/PositiveStringNumber.java

```diff
@@ -0,0 +1,19 @@
+package calculator;
+
+public class PositiveStringNumber {
+    int number;
+
+    public PositiveStringNumber(String input) {
+        int intNumber = parseInt(input);
+        if (intNumber < 0) throw new RuntimeException("음수는 들어올 수 없습니다");
+        this.number = intNumber;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public static int parseInt(String s) throws NumberFormatException {
```

Comment:
외부에서 사용되지 않을 것 같은데 `private`으로 접근 제한자를 변경하는건 어떨까요?

그리고 NumberFormatException 발생시에 대한 처리가 따로 없다면 `throws NumberFormatException`가 제거되어도 괜찮을 것 같습니다.

```suggestion
    private int parseInt(String s) {    
```